personality_insights/ApplicationBuilder/personality-insights-twitter-widget.erb

```diff
@@ -0,0 +1,108 @@
+
+<script>
+  
+  function displayModel(model) {
+    var $model_container = $(".user_model").empty(); 
+    
+    // model_container.append(JSON.stringify(model));
```

Comment:
This looks like debugging that we can remove altogether instead of commenting out.